jsonnet/prometheus.jsonnet

```diff
@@ -114,35 +114,7 @@ local namespacesRole =
 
     // OpenShift has the kube-apiserver as well as an aggregated API called
```

Comment:
Is this comment still relevant?

Reply:
No, I will remove it in https://github.com/openshift/cluster-monitoring-operator/pull/438